### DISCUSSION TITLE
Build failure on Debian Jessie

### DISCUSSION THREAD

**Discussion by davidl2** [magefile/mage#439] [Q&A] [answered]
Of commit 26cdb5c7369a9b03981eb906dd3606c566e45c16 from git...
```
$ go version
go version go1.3.3 linux/386
$ go run bootstrap.go 
../../go/src/github.com/magefile/mage/mg/deps.go:4:2: cannot find package "context" in any of:
        /usr/lib/go/src/pkg/context (from $GOROOT)
        /home/dllamber/go/src/context (from $GOPATH)
```

Is a certain minimum version of go required to build? Or is a 64-bit platform required?

**Comment by natefinch** [comment, marked as answer]
Uh, well, go 1.3 is reaaaally old. go 1.3 was released in 2014.

The context package, which is what that error is complaining about, was introduced in go 1.7, which was released in 2016. It's used in almost every go product at this point. 

You need to update your installed version of go.

Luckily, this is really safe, as there are almost never breaking changes in new versions of Go (although, upgrading 16 versions might have a small chance of breaking something).
